modules/nixos.nix

```diff
@@ -70,11 +71,13 @@
 
         clobber = mkOption {
           type = bool;
-          default = false;
-          example = true;
+          default = cfg.clobberByDefault;
+          defaultText = literalMD "Set to the value of {option}`hjem.clobberByDefault`";
```

Comment:
```suggestion
          defaultText = literalExpression "config.hjem.clobberByDefault";
```

Reply:
This omits `{option}` formatting iirc, haven't tested though.